app/controllers/PdfDownloadController.scala

```diff
@@ -42,10 +42,20 @@ class PdfDownloadController @Inject()(appConfig: FrontendAppConfig,
         pdfService.generatePdf(applicationPdf(appConfig, c, attachmentData)).map { binaryFile =>
           Results.Ok(binaryFile.content)
             .as(binaryFile.contentType)
-            .withHeaders(CONTENT_DISPOSITION -> s"filename=confirmation_$reference.pdf")
+            .withHeaders(CONTENT_DISPOSITION -> s"filename=BTIConfirmation$reference.pdf")
         }
       }
     }
   }
 
```

Comment:
The two methods in this class are very similar.
You could create a common private method used by both (DRY principle)

Reply:
Pulled out the common code into a private method